### ISSUE TITLE
Latest stable version? 1.5.4 or 1.6?

### ISSUE THREAD

**Issue by ilovezfs** [hrydgard/ppsspp#11073]
https://github.com/hrydgard/ppsspp/releases has one opinion

https://github.com/hrydgard/ppsspp/releases/latest has another

**Comment by ilovezfs** [comment]
@LunaMoo we need to know when and whether to update the formula in Homebrew.

**Comment by hrydgard** [comment]
So? You don't have to open issues *during* the release. Next time have some patience.

**Comment by ilovezfs** [comment]
I don't think you understand. I have no idea when your release process is going on, only that there was a discrepancy between the most recent tag and the most recent release. This is a fairly common scenario that can go on for weeks in some cases.

**Comment by hrydgard** [comment]
Here, it doesn't go on for weeks, at most half an hour maybe. I guess either you had some automatic monitoring of this or you got really unlucky with your timing. The issue is fixed, anyhow.

**Comment by ilovezfs** [comment]
>I guess either you had some automatic monitoring of this or you got really unlucky with your timing.

We run `brew livecheck` a few times per day and check for any updates. It just happened that one of those runs was in the middle of your release process.

In any case, I've now shipped the latest version in Homebrew:
https://github.com/Homebrew/homebrew-core/pull/28307
